src/libraries/System.Net.NetworkInformation/src/System/Net/NetworkInformation/NetworkAddressChange.UnknownUnix.cs

```diff
@@ -9,15 +9,6 @@ namespace System.Net.NetworkInformation
 {
     public partial class NetworkChange
     {
-        static NetworkChange()
-        {
-            // fake usage of static readonly fields to avoid getting CA1823 warning when we are compiling this partial.
```

Comment:
Is this not accurate?

Reply:
It was but I changed csproj file not to pull these private fields at all